### ISSUE TITLE
gmail right side ads still showing

### ISSUE THREAD

**Issue by patrickdo** [uBlock-LLC/uBlock#203]

**Comment by gorhill** [comment]
I can't see ads.
1. Browser version?
2. Is it when launching the browser?
3. What are the ads?
4. Is it on `mail.google.com`?
5. What filter lists are selected?
6. What was the block count indicated by uBlock?
7. Screenshot would be great.

**Comment by patrickdo** [comment]
- see "samsung at best buy" ad on the right
- chrome: Version 39.0.2138.3 dev-m
- µBlock 0.5.3.0
- when i'm on mail.google.com

**Comment by gorhill** [comment]
I can't investigate if I can't reproduce. I just tried with Google Chrome 39 dev, and still can't see any ads. Can you forward me the email for which the ad is showing? More details is better when it comes to investigate issues. Still don't know about 2, 5, 6.

**Comment by pwstokes225** [comment]
I also see ads appear to the right or below when viewing emails, but only after leaving the gmail tab open for a long time. If I refresh the gmail tab, or open a fresh one, no ads are present.

To answer your questions:
1. 39.0.2138.3 dev-m (though this has happened on previous versions too).
2. No.
4. Yes.
5. Everything that isn't redundant and isn't in "Regions, languages".
3,6,7. As I have now refreshed it, I will probably have to wait a day or so to reproduce.

**Comment by kurtextrem** [comment]
Could it be that the gmail tab was pinned (= open after starting chrome)?

**Comment by pwstokes225** [comment]
That's probably the reason, actually. Pinned gmail has ads on startup.
It seems the reason why I rarely saw gmail ads was because I rarely restarted Chrome.

**Comment by kurtextrem** [comment]
Known already: https://github.com/gorhill/uBlock/issues/184

**Comment by gorhill** [comment]
> after leaving the gmail tab open for a long time

Also, note that if uBlock itself updated after the tab was opened, this could causes ads to start appearing, as the content script of the older version of uBlock would still be used, but no longer connected to the extension.

**Comment by ghost** [comment]
:+1: to that https://github.com/gorhill/uBlock/issues/203#issuecomment-54313836
for that was the case for me, it seems, just refreshing the tab  hid the ad below a g+ email notification about a comment.

**Comment by gorhill** [comment]
With v 0.6.0.0, ads will be hidden even when the web page is opened before uBlock is launched. But unsure whether the problem here was caused by this.

**Comment by gorhill** [comment]
This issue share two things with issue #223:
- Ads in Gmail
- Chrome 39 dev channel

I will mark this one as duplicate of #223 as there is more details in there.
